### PR TITLE
Combined batch edge case: Avoid a number of batches audited count greater than zero to start

### PR DESCRIPTION
**Pull request by arsalansufi**
## Overview

Closes https://github.com/votingworks/arlo/issues/2118

If you create a combined batch with at least 2 sampled batches, before any batches have been audited, the audit admin UI will indicate that at least 1 batch has already been audited. This is because we persist a "zero" tally sheet on combined batch creation for all sub batches other than the representative batch.

This PR solves this problem by moving that zero writing to batch tally entry, i.e., when data for the representative batch is entered.

<table>
edge-case
</table>

## Testing

- [x] Tested manually
- [x] Updated automated tests

### CODE REVIEW

**Review by jonahkagan — APPROVED** [review]
✅

File: server/api/support.py [hunk 1]
```diff
@@ -704,27 +703,8 @@ def create_combined_batch(jurisdiction_id: str):
...
-                    results=[
-                        BatchResult(contest_choice_id=choice.id, result=0)
-                        for contest in list(jurisdiction.contests)
-                        for choice in contest.choices
-                    ],
-                )
-            ]
-            batch.last_edited_by_support_user_email = support_user_email
-            batch.last_edited_by_user_id = None
-            batch.last_edited_by_tally_entry_user_id = None
```
[the remark this answers is not in the document]
Comment (arsalansufi) [reply server/api/support.py]: My general mental model has been that we use these "last edited by" fields to capture when tallies are entered for a batch and nothing else. That's the only place in the UI that we use these fields at least, in the footer of the tally entry UI to indicate who entered in the tallies. So I've kept tally population and "last edited by" field population coupled, moving both from combined batch creation to representative batch tally entry.
